test/views/behaviours/ViewBehaviours.scala

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.behaviours
+
+import base.SpecBase
+import org.jsoup.Jsoup
+import org.jsoup.nodes.{Document, Element}
+import org.scalatest.Assertion
+import play.twirl.api.HtmlFormat
+import views.base.ViewSpecAssertions
+
+import scala.collection.JavaConverters._
+
+trait ViewBehaviours extends SpecBase with ViewSpecAssertions {
+
+  def view: HtmlFormat.Appendable
+
+  def parseView(view: HtmlFormat.Appendable): Document = Jsoup.parse(view.toString())
+
+  lazy val doc: Document = parseView(view)
+
+  val prefix: String
+
+  val hasSignOutLink: Boolean = true
+
+  val urlContainsLrn: Boolean = false
+
+  if (hasSignOutLink) {
+    "must render sign out link in header" in {
+      val link = getElementByClass(doc, "hmrc-sign-out-nav__link")
+      assertElementContainsText(link, "Sign out")
+      assertElementContainsHref(
+        link,
+        "http://localhost:9553/bas-gateway/sign-out-without-state?continue=http://localhost:9514/feedback/manage-transit-departures"
+      )
+    }
+
+    "must render timeout dialog" in {
+      val metas = getElementsByTag(doc, "meta")
+      assertElementExists(metas, _.attr("name") == "hmrc-timeout-dialog")
+      if (urlContainsLrn) {
+        //assertElementExists(metas, _.attr("data-keep-alive-url") == s"/manage-transit-movements/arrival/$lrn/keep-alive")
+      } else {
+        //assertElementExists(metas, _.attr("data-keep-alive-url") == "/manage-transit-movements/arrival/keep-alive")
```

Comment:
I see the lrn has been incorporated into MainTemplate - can these lines be uncommented?

Reply:
Ahhh, I knew there was something I was supposed to do.